src/elements/checkboxes/_checkboxes.scss

```diff
@@ -0,0 +1,45 @@
+/**
+ * Checkboxes
+ */
+
+// Dependencies
+@import 'config/variables';
+@import 'config/checkboxes';
+@import 'config/direction';
+@import 'config/fonts';
+
+// Declarations
+.checkbox {
+  font-size: 16px;
+  font-weight: $font-weight-bold;
+  display: block;
+
+  margin-bottom: $spacing-base;
```

Comment:
@kimpenguin the $spacing-base is a holdover from the original ACCESS stylesheet. I was hoping we could avoid it in NYCO Patterns. Any instance of $spacing-base could just be $grid (8px) or a desired multiple of it ($grid *2). I'm in the process of fixing this on ACCESS Patterns but I couldn't do it in one sweep.